Script/Player.gd

```diff
@@ -3,6 +3,11 @@ extends CharacterBody2D
 signal stomped(goober: CharacterBody2D)
 signal died
 
+## If true, the player cannot be moved and does not interact with its
+## environment; it just performs its idle animation. Set this on title & win
+## screens.
+@export var inert := false
```

Comment:
Considering the description above, what about `interactive = false` ? Or `is_prop = true`. In the  "theatrical property" sense (also used in game dev), a prop is something that is on stage, and is not the actors or the scenery.

Reply:
"Interactive" sounds good to me. I'll rename & flip the sense of the flag.

I do like the "prop" concept but I fear it's one more thing to explain and also is a bit of a namespace collision with "prop(erty)" in the programming sense.

Reply:
I made this change and reworded the commit messages accordingly.